### DISCUSSION TITLE
What would a blog look like if we started using it in the current website?

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/website#53] [Ideas]
## Video of what it looks like enabling the blog in the current state of the site
We gain all of the below video for free today with very little effort. (note the video was done in a local environment)

https://user-images.githubusercontent.com/8985705/158720140-dc9bd7b8-6bba-4cef-b5c0-042ba263969e.mov

## Some considerations
- Need to update button styling for comments
- do we want to use comments? Anti spam features can be implemented at no cost.
- Do we want a grid layout instead of the vertical blog roll? (Let's try to keep this design very minimal as it needs to repeat as an archive)

**Comment by mrmetaverse** [comment]
bump. 
###### note: With the recent workshops, we have been missing bringing up some of these items in our weekly agendas. I think this is a fun one to open up to the community during a Wednesday call, and to get on the next website working group agenda.

Thanks for putting together this demo. This is great work. It feels on-brand and aligned with all of our other new designs (gotta love a CMS). Overall, I think the blog roll layout is clean, and modern. I love the simplicity! 

RE: Button styling for comments. Is this a chance for a fancy micro-interaction? :eyes:  @jsejms

**Comment by mrmetaverse** [comment]
Created this milestone for tracking blog fodder and content strategy related to our website: https://github.com/omigroup/website/milestone/2

**Comment by mrmetaverse** [comment]
related: https://github.com/omigroup/omigroup/issues/205 Shirley and I met to discuss some of the ways that she could get involved.
